### PR TITLE
Took out all emoji in pages.

### PR DESCRIPTION
**Pull request by Bangalisch**
Emoji's might not always show properly and they are used inconsistently. After talks with pavlenex, decided to clean up all the pages and stick to no emoji's

### CODE REVIEW

**Comment by pavlenex** [comment]
@Bangalisch thanks for taking care of this. It seems that the build is failing, it seems there's a broken link somewhere. Can you address that one?

```
Getting links from: http://localhost:8080/Deployment/LunaNode/
[48](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:48)
Getting links from: http://localhost:8080/Deployment/Azure
[49](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:49)
Getting links from: http://localhost:8080/Deployment/GoogleCloud/
[50](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:50)
Getting links from: http://localhost:8080/Deployment/Hardware
[51](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:51)
Getting links from: http://localhost:8080/Deployment/RaspberryPi
[52](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:52)
Getting links from: http://localhost:8080/Deployment/Hack0/
[53](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:53)
Getting links from: http://localhost:8080/Deployment/ManualDeployment/
[54](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:54)
Getting links from: http://localhost:8080/Deployment/ManualDeploymentExtended/
[55](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:55)
├─BROKEN─ http://localhost:8080/Deployment/FAQ/ (HTTP_404)
[56](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:56)
Getting links from: http://localhost:8080/Deployment/Azure/
[57](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:57)
Getting links from: http://localhost:8080/FAQ/Deployment/#what-are-the-minimal-requirements-for-btcpay
[58](https://github.com/btcpayserver/btcpayserver-doc/runs/5964507697?check_suite_focus=true#step:10:58)
Getting links from: http://localhost:8080/Deployment/RaspberryPi/
```

**Comment by Bangalisch** [comment]
ty @pavlenex seems good now :) ?

File: docs/Deployment/ManualDeploymentExtended.md [hunk 1]
```diff
@@ -1300,4 +1300,8 @@ Updating could break things. Be careful on a live system.
 ~$ sudo systemctl start rtl
 ```
 
-## 🏁 The End
+##  The End
+
+### Questions
+
+Join the [community chat](https://chat.btcpayserver.org/) on Mattermost by downloading [Mattermost app](https://mattermost.com/download/), or on [Telegram](https://t.me/btcpayserver) in case you need further help or help or want to hang around with like-minded people.
```
Comment (pavlenex) [inline docs/Deployment/ManualDeploymentExtended.md:1307]: Why is this needed on every page you document? 😄

Comment (Bangalisch) [reply docs/Deployment/ManualDeploymentExtended.md:1307]: I've mainly touched bigger topic pages. 
If you look at Dashboard's PR, you will find I did not put it there. 
As that is a straight forward page, this extended manual Deployment might raise questions by user. :)

**Review by pavlenex — APPROVED** [review]
Let's go Bas🎉
